src/store/components/AdminItemPage/index.tsx

```diff
@@ -145,7 +183,7 @@ const AdminItemPage: React.FC<AdminItemPageProps> = (props) => {
 
             if (values.newPicture) {
               const formdata = new FormData();
-              formdata.append('image', (values.newPicture as unknown) as Blob);
+              formdata.append('image', values.newPicture);
```

Comment:
By making `values.newPicture` the type `Blob`, this typing issue has been resolved.